go.mod

```diff
@@ -5,6 +5,7 @@ require (
 	github.com/Azure/azure-storage-blob-go v0.0.0-20181022225951-5152f14ace1c
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da
+	github.com/cespare/xxhash v1.1.0
```

Comment:
perfect choice :+1:

let's document that we're using this hash in the design doc/proposal

Reply:
this is still to be addressed

Reply:
I looked back, and it was actually already included it in the proposal: https://github.com/improbable-eng/thanos/commit/64e7a2607392289bec5d39b9c40fb6c261eed9ba#diff-4413dcb353f516890e28f066b0e56b90R96